bin/check-http.rb

```diff
@@ -201,7 +201,7 @@ def run
     end
 
     begin
-      Timeout.timeout(config[:timeout]) do
+      Timeout.timeout(config[:timeout] + 10) do
```

Comment:
I must be missing something, it sounds like either we have a bad default (which I categorize as does not work for most users and for a majority of use cases) and should modify that or you need to configure `--timeout` for your use case. Depending on what you are asking it to do (this check does lots of things) the time will be different. A sane default is just that a default and you can override as you see fit. Originally I was gonna propose a new option for this but the more I stare at this I feel like you just need to set your timeout based on your use case which depends on what you ask it to do and the resources available for execution.